app/controllers/posts_controller.rb

```diff
@@ -4,6 +4,28 @@ def index
     @posts = @user.posts
   end
 
+  def new
+    @post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: @post } }
+    end
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
```

Comment:
You can create the private `post_params` and use it here similar to comments_params. 

```ruby
def post_params
      params.require(:post).permit(:title, :text)
    end
```